htdocs/workstation/workstation_note.php

```diff
@@ -53,7 +54,7 @@
 // Load object
 include DOL_DOCUMENT_ROOT.'/core/actions_fetchobject.inc.php';     // Must be include, not include_once. Include fetch and fetch_thirdparty but not fetch_optionals
 if ($id > 0 || !empty($ref)) {
-	$upload_dir = $conf->workstation->multidir_output[$object->entity]."/".$object->id;
+	$upload_dir = $conf->workstation->multidir_output[$object->entity ? $object->entity : $conf->entity]."/workstation/".get_exdir(0, 0, 0, 1, $object);
```

Comment:
Are you sure you must add /workstation/ as subdir ? The ->workstation->multidir_output shouldalready contains the dir nameof the module.

Reply:
I take the line of code in workstation_document.php and yes it give `documents/workstation/workstation`